gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/AbstractBndrun.java

```diff
@@ -109,12 +112,26 @@ public RegularFileProperty getBndrun() {
 	 * @return The bundles to be added to a FileSetRepository for non-Bnd
 	 *         Workspace builds.
 	 */
-	@InputFiles
-	@PathSensitive(NONE)
+	@Internal
 	public ConfigurableFileCollection getBundles() {
 		return bundles;
 	}
 
+	/**
+	 * Wrapper of the {@link #getBundles()} method to allow <code>@Classpath</code> normalization and sorting.
+	 *
+	 * @return The sorted bundles.
+	 */
+	@Classpath
+	public Provider<List<File>> getBundlesSorted() {
```

Comment:
Can we reduce the accessibility from `public`? `private` would be preferable if Gradle will use this. This avoids this method becoming API.

Reply:
If we cannot reduce the accessibility from public, then we should note in the javadoc that this method in not API and is only used by Gradle to access getBundles in a sorted manner for normalization purposes.